src/Components/Book/Book.test.js

```diff
@@ -1,3 +1,65 @@
-import React from 'react';
-import { screen, render, waitFor } from '@testing-library/react';
+import React from 'react'
+import '@testing-library/jest-dom'
+import { screen, render, waitFor, fireEvent } from '@testing-library/react'
 import Book from './Book'
+import { MemoryRouter } from 'react-router-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import { rootReducer } from '../../reducers';
+
+describe('Book Component', () => {
+  it('should render book information',() => {
+    const book1 = {
+      rank: 1 ,
+      primary_isbn10: "1250145236",
+      title: "ALL THE DEVILS ARE HERE",
+      book_image: "https://s1.nyt.com/du/books/images/9781250145239.jpg",
+      author: "Louise Penny"
+    }
+    const store = createStore(rootReducer);
+
+    render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <Book book={book1}/>
+        </MemoryRouter>
+      </Provider>
+    )
+
+    const bookRank = screen.getByText("1", { exact: false })
```

Comment:
Nice work using { exact: false } for the rank! I'll have to add this into one of my tests as I couldn't work out how to find the element!